### PR TITLE
Remove getpass add prompt_toolkit

### PR DESCRIPTION
**Pull request by Notnaton**
### Pre-Submission Checklist (optional but appreciated):

- [ ] I have included relevant documentation updates (stored in /docs)
- [ ] I have read `docs/CONTRIBUTING.md`
- [ ] I have read `docs/ROADMAP.md`

### OS Tests (optional but appreciated):

- [ ] Tested on Windows
- [ ] Tested on MacOS
- [ ] Tested on Linux

### CODE REVIEW

File: interpreter/core/computer/ai/ai.py [hunk 1]
```diff
@@ -106,9 +106,10 @@ def query(self, text, query, custom_reduce_query=None):
         # Retrieve OpenAI API key
         openai_api_key = os.getenv("OPENAI_API_KEY")
         if openai_api_key is None:
-            openai_api_key = getpass.getpass(
-                prompt="Please enter your OpenAI API key: "
+            openai_api_key = prompt(
+                message="Please enter your OpenAI API key: ", is_password=True
```
Comment (Arrendy) [inline interpreter/core/computer/ai/ai.py:110]: Does it need the `message` key?
 https://python-prompt-toolkit.readthedocs.io/en/master/pages/asking_for_input.html#password-input

Comment (Notnaton) [reply interpreter/core/computer/ai/ai.py:110]: ¨no

**Review by Arrendy — APPROVED** [review]

**Comment by KillianLucas** [comment]
Nice @Notnaton!! That behavior of getpass is so weird and confusing. I much, much prefer this. Merging!

**Comment by KillianLucas** [comment]
Also thanks @Arrendy for the review!

**Pull request merged**